assets/css/variebles.css

```diff
@@ -0,0 +1,15 @@
+:root {
+  --primary-font: "open-sans";
+  --thin-font: 300;
+  --normal-font: 400;
+  --bold-font: 600;
+
+  --background-color: #112b3c;
+  --primary-color: #f66b0e;
+  --secondery-color: #a5c9ca;
+  --border: 1px solid #f66b0e;
+
+  --xSmall: 0.5em;
+  --small: 0.9em;
+  --Large: 3em;
```

Comment:
Would you please follow a same pattern across variable namings?
`--my-name` or `--MyName` or `--myName`

Reply:
Done.